Tests/scripts/collect_tests_and_content_packs.py

```diff
@@ -1042,7 +1042,13 @@ def get_test_list_and_content_packs_to_install(files_string, branch_name, minimu
 
     from_version, to_version = get_from_version_and_to_version_bounderies(all_modified_files_paths, id_set)
 
-    create_filter_envs_file(from_version, to_version)
+    # Check if only README file in file string, if so, no need to create the servers.
+    files = [s for s in files_string.split('\n') if s]
+    readme = all(map(lambda s: 'README.md' in s, files))
+    if readme:
+        create_filter_envs_file(from_version, to_version, readme_only=True)
```

Comment:
no need to call `create_filter_envs_file` twice, just build the args dict and pass them to the func